src/app/components/FrostedGlassPromo/index.jsx

```diff
@@ -15,8 +15,40 @@ import ImageWithPlaceholder from '../../containers/ImageWithPlaceholder';
 
 import withData from './withData';
 
-const Body = styled.div`
+const Wrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  max-width: 400px;
+  text-decoration: none;
+  &:hover {
```

Comment:
Missing focus?

Reply:
This is a div so will never have focus - there is focus styling on the `a`: https://github.com/bbc/simorgh/blob/801985b1efa1162255e4ec965b496360fca5fa52/src/app/components/FrostedGlassPromo/index.jsx#L61